jx.hcl

```diff
@@ -0,0 +1,11 @@
+description = "Jenkins X CLI"
+test = "jx version"
+binaries = ["jx"]
+version "3.2.137" {
+  linux {
```

Comment:
These blocks can be pulled out to the top level if you prefer, then new versions can be added with just a single string, eg.

```hcl
version "3.2.137" "3.2.138" {}
```

Maybe roll  it into the next update! :)

Reply:
@alecthomas ah yeah! I went via the docs example, but yeah looking at more other examples it does look like lots of packages do things that way. Lemme fix...